src/main/java/org/sonarsource/auth/github/GitHubIdentityProvider.java

```diff
@@ -85,7 +85,7 @@ public boolean allowsUsersToSignUp() {
   public void init(InitContext context) {
     String state = context.generateCsrfState();
     OAuthService scribe = prepareScribe(context)
-      .scope("user:email")
+      .scope("user:email,read:org")
```

Comment:
The scope `read:org` should be added only if the organization filter is defined.